components/expenses/DownloadInvoicesPopOver.js

```diff
@@ -149,10 +149,13 @@ class Overlay extends React.Component {
     }
     const invoices = data.allInvoices;
     const years = uniq(invoices.map(i => i.year));
-    const months = uniq(invoices.filter(i => Number(i.year) === Number(this.state.year)).map(i => i.month));
+    const months = []; //uniq(invoices.filter(i => Number(i.year) === Number(this.state.year)).map(i => i.month));
```

Comment:
@snitin315 seems like a leftover from testing?

```suggestion
    const months = uniq(invoices.filter(i => Number(i.year) === Number(this.state.year)).map(i => i.month));
```

Reply:
Oh yes! My bad :-). Thanks for pointing.